.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+
+python:
+- 2.7
+
+install:
+- make install
+
+script:
+- make check
```

Comment:
I think the `check` target should/will depend on the `test` target. So I'd just that `check` is the target we run on the CI system.